### PR TITLE
Convert scopes to callable

### LINKED ISSUE #473: Ensure scopes use callable objects
_From @benjaminwood on April 12, 2018 4:37_

> Rails 4.0 requires that scopes use a callable object such as a Proc or lambda:

```ruby
scope :active, where(active: true) 

# becomesscope

:active, -> { where active: true }
```

Quite a few scopes will need to be updated.

_Copied from original issue: hintmedia/rigse#21_

### PR DESCRIPTION
**Pull request by tjchambers**
Fixes #473 in advance of Rails 4.0 upgrade where scopes must be callable objects.

### CODE REVIEW

File: app/models/commons_license.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ class CommonsLicense < ActiveRecord::Base
   DeedFormat  =   "http://#{Site}/licenses/%{code}/%{version}/"
   LegalFormat =   "http://#{Site}/licenses/%{code}/%{version}/legalcode"
 
-  default_scope :order => 'number ASC'
+  default_scope { order('number ASC') }
```
Comment (benjaminwood) [inline app/models/commons_license.rb]: Missing `->` here?

Comment (benjaminwood) [reply app/models/commons_license.rb]: I see other `default_scope` are missing `->`. I'm guessing this is intentional. :smile:

Comment (tjchambers) [reply app/models/commons_license.rb]: default scopes do not need them. They are implied I believe.

**Comment by tjchambers** [comment]
This is green except for the 3 cucumber errors inherited from master. I am going to use this after merged to start work on #466 since some of the cleanup was done in this `scope` focused change.

**Comment by scytacki** [comment]
I wish I could see the code climate diff stats, but hopefully that will start working once we fix the failing cucumber tests.

**Pull request merged**